playbooks/provisioning/openstack/README.md

```diff
@@ -325,7 +325,23 @@ if requested, and DNS server, and ensures other OpenShift requirements to be met
 
 ### Running Custom Post-Provision Actions
 
-If you'd like to run post-provision actions, you can do so by creating a custom playbook. Here's one example that adds additional YUM repositories:
+A library of custom post-provision actions exists in `openshift-ansible-contrib/playbooks/provision/openstack/custom-actions`. Playbooks include:
+
+  - add-yum-repos.yml
+
+A custom playbook can be run like this:
+
+```
+ansible-playbook --private-key ~/.ssh/openshift -i myinventory/ openshift-ansible-contrib/playbooks/provision/openstack/custom-actions/custom-playbook.yml
```

Comment:
We're using `inventory` instead of `myinventory` in this document. Could you please update that, too?

Reply:
Done!